spec/models/fcrepo_endpoint_spec.rb

```diff
@@ -1,31 +1,43 @@
 # frozen_string_literal: true
 
 RSpec.describe FcrepoEndpoint do
+  subject { described_class.new base_path: base_path }
+
   let(:base_path) { 'foobar' }
 
   describe '.options' do
-    subject { described_class.new base_path: base_path }
-
     it 'uses the configured application settings' do
       expect(subject.options[:base_path]).to eq base_path
     end
   end
 
   describe '#ping' do
-    let(:success_response) { double(response: double(success?: true)) }
+    let(:url_prefix) { ActiveFedora::Fedora.instance.connection.connection.http.url_prefix.to_s }
+
+    context 'when Fedora is accessible' do
+      let(:success_response) { double(response: double(success?: true)) }
+
+      it 'returns "Fedora is OK"' do
+        allow(ActiveFedora::Fedora.instance.connection).to receive(:head).with(url_prefix).and_return(success_response)
+        expect(subject.ping).to eq("Fedora is OK")
+      end
+    end
+
+    context 'when Fedora is down' do
+      let(:failure_response) { double(response: double(success?: false)) }
 
-    it 'checks if the service is up' do
-      allow(ActiveFedora::Fedora.instance.connection).to receive(:head).with(
-        ActiveFedora::Fedora.instance.connection.connection.http.url_prefix.to_s
-      ).and_return(success_response)
-      expect(subject.ping).to be_truthy
+      it 'returns "Fedora is Down"' do
+        allow(ActiveFedora::Fedora.instance.connection).to receive(:head).with(url_prefix).and_return(failure_response)
+        expect(subject.ping).to eq("Fedora is Down")
```

Comment:
Fixing the spec to reflect my wondering:

```suggestion
        expect(subject.ping).to eq("Error checking Fedora status: Fedora is Down")
```